prompts/template_xml/context.txt

```diff
@@ -1,8 +1,21 @@
-The function is defined in
-<include_file>
-{CONTEXT_HEADER}
-</include_file>
+The code block below contains the header files which you must include. The function requires these header files to successfully compile.
+If the headers are private, do not include them.
+Header files:
+<code>
+{CONTEXT_HEADERS}
+</code>
+
+You must insert code in the below code block before the function being tested:
+<code>
+{CONTEXT_MUST_INSERT}
+</code>
+
+Here is the source code of the function being tested:
+<code>
+{CONTEXT_FUNC_SOURCE}
+</code>
 
+Here is the source code for functions which reference the function being tested:
```

Comment:
Can we omit these sections if any of them are empty? Instead of just having an empty code block? 

I wonder if that's impacting the efficacy of this.

Reply:
I did this in https://github.com/google/oss-fuzz-gen/pull/144/commits/5ac4932e7924a049c0b6b6a48ff32c2577f0bb9f